lib/compat/wordpress-5.9/theme-templates.php

```diff
@@ -69,7 +69,7 @@ function gutenberg_filter_wp_template_unique_post_slug( $override_slug, $slug, $
 	return $override_slug;
 }
 
-// Remove 5.8 filter if existant.
+// Remove 5.8 filter if existent.
```

Comment:
I think it supposed to be "exists" here.

Reply:
Fixed in latest commit.

Reply:
Thank you!